src/components/blocks/blocks.jsx

```diff
@@ -1,25 +1,21 @@
 const React = require('react');
-
+const Box = require('../box/box.jsx');
 const styles = require('./blocks.css');
 
-class BlocksComponent extends React.Component {
-    render () {
-        const {
-            componentRef,
-            ...props
-        } = this.props;
-        return (
-            <div
-                className={styles.blocks}
-                ref={componentRef}
-                {...props}
-            />
-        );
-    }
-}
-
+const BlocksComponent = props => {
+    const {
+        componentRef,
+        ...componentProps
+    } = props;
+    return (
+        <Box
+            className={styles.blocks}
+            componentRef={componentRef}
+            {...componentProps}
+        />
+    );
+};
```

Comment:
This came from trying to adhere to the [react/prefer-stateless-function](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prefer-stateless-function.md) warning, which in turn exists because the React team has suggested that stateless components written this way will receive performance enhancements: https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#stateless-functional-components in addition to other nice things about them being functions: https://medium.com/@housecor/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc